.github/workflows/build-and-publish.yml

```diff
@@ -25,6 +25,12 @@ jobs:
       - name: Build Script Language Container (SLC)
         run: poetry run nox -s build_language_container
 
+      - name: PyPi Release
+        env:
+          POETRY_HTTP_BASIC_PYPI_USERNAME: "__token__"
+          POETRY_HTTP_BASIC_PYPI_PASSWORD: "${{ secrets.PYPI_TOKEN }}"
+        run: poetry publish
```

Comment:
Is this part of the bugfix?

Reply:
You are right, additionally (as part of the clean-up after the release) I moved the pypi release back to the old position in the workflow, which is before the GitHub release. 
I hope, this is OK for you?

Reply:
But I am not ok with it, because a PyPi release doesn't make sense, if we couldn't upload the SLC